src/Equinor.ProCoSys.IPO.Infrastructure/Repositories/DapperRepositoryBase.cs

```diff
@@ -10,13 +10,13 @@ namespace Equinor.ProCoSys.IPO.Infrastructure.Repositories;
 
 public class DapperRepositoryBase
 {
-    private readonly IPOContext _context;
+    public readonly IPOContext Context;
```

Comment:
Add a comment why this should be private, but we need to be make it public. Start the comment with phrase // HACK